packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -285,7 +285,7 @@ export class DefaultPageBlockComponent extends LitElement implements BlockHost {
     });
 
     // TMP: clear selected rects on scroll
-    document.addEventListener('wheel', this._clearSelection);
+    document.addEventListener('wheel', this._clearSelection.bind(this));
```

Comment:
I think the `_clearSelection` is refactored so `.bind` is no longer needed. Also, we should try to avoid `.call`/`.apply`/`.bind` in our codebase.

Reply:
OK!

Reply:
this PR is the way of fixing `this` 

https://github.com/toeverything/blocksuite/commit/9f9739d75cc19afbdfd97b7d991f44722a4d3907